backend/locale/de/LC_MESSAGES/django.po

```diff
@@ -431,37 +431,35 @@ msgstr "Englisch"
 msgid "German"
 msgstr "Deutsch"
 
-msgid ""
-"\n"
-"        404 Error\n"
-"    "
-msgstr ""
-"\n"
-"        404 Error\n"
-"    "
+msgid "403 Error"
+msgstr "403 Error"
 
-msgid "This page could not be found."
-msgstr "Diese Seite konnte nicht gefunden werden."
+msgid "You don't have the permissions to access this page."
```

Comment:
To be picky, if we want to be consistent, the page should be in German. We could also open an issue to switch German -> English, to be easier usable in an international context

Reply:
We discussed that we want to slowly transition to purely english source pages, so that's fine.